### PR TITLE
Add SECURITY.md

### PR DESCRIPTION
**Pull request by dmcgowan**
Adds process for reporting security vulnerabilities and joining the security announce mailing list.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a containerd vulnerability, send an email to `security@containerd.io`
+detailing the issue and steps to reproduce. The reporter(s) can expect a
+response within 24 hours acknowledging the issue was received. If a response is
+not received within 24 hours, please reach out directly to any maintainer
+directly to confirm receipt of the issue.
```
Comment (mikebrow) [inline SECURITY.md:9]: r/directly to any maintainer directly/to any maintainer directly/

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Review Process
+
+Once a maintainer has confirmed the relevance of the report, a draft security
+advisory will be created on Github. The advisory will be used to discuss the
```
Comment (mikebrow) [inline SECURITY.md:14]: r/advisory/draft advisory/

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+issue with maintainers, the reporter(s), and containerd's security advisors. If
+the reporter(s) wishes to participate in this discussion, then provide reporter
+Github username(s) to be invited to the discussion. If the reporter(s) does not
+wish to participate directly in the discussion, then the reporter(s) can request
+to be updated regularly via email.
+
+If the vulnerability is accepted, a timeline for developing a patch, public
+disclosure, and patch release will be determined. If there is an embargo period
+on public disclosure before the release, an announcment will be sent to the
```
Comment (mikebrow) [inline SECURITY.md:23]: r/release/patch release/

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+security announce mailing list (`containerd-security-announce@lists.cncf.io`)
+announcing the scope of the vulnerability, the date of availability of the
+patch, and the date of public disclosure. The reporter(s) are expected to
```
Comment (mikebrow) [inline SECURITY.md:26]: r/patch/patch release/

**Review by fuweid — APPROVED** [review]
LGTM

**Comment by mikebrow** [comment]
LGTM

**Pull request merged**